core/src/edu/rice/comp413fall2020grey/Location/registrar/ConnectionsManager.java

```diff
@@ -0,0 +1,13 @@
+package edu.rice.comp413fall2020grey.Location.registrar;
+
+
+import java.util.List;
+import java.util.UUID;
+
+public interface ConnectionsManager {
+
+    String getAddressByServerUUID(UUID serverUUID);
+
+    List<String> getClientsByServerUUID(UUID serverUUID);
+    
+}
```

Comment:
@yopyop1007  This looks like it has a lot of overlap with networking -- does this conflict with anything you have?